elastic/kibana
# [Actionable Observability] Move the alerts search bar to triggers action UI plugin

Resolves #144393

## Summary

In this PR, I just move the alerts search bar component from the observability plugin to the triggers action UI. There should not be any change functionality-wise.

## How to test
- Go to the alerts page in observability
- Make sure that the alerts search bar works as before
